### ISSUE TITLE
check path compatibility with Windows

### ISSUE THREAD

**Issue by quantumjot** [lowe-lab-ucl/segment-classify-track#14]
In several places, paths are constructed using f-strings. We might need to check that these work on Windows.  Better to use [pathlib](https://docs.python.org/3/library/pathlib.html) (or `os.path.join`) to ensure cross-platform compatibility.

**Comment by nthndy** [comment]
yeah i ended up testing a bit of this on friday with manasi and you are correct! will wrap all f-strings in the `Path` function from pathlib.
